scripts/upload-counterpart-agreements.py

```diff
@@ -42,10 +42,11 @@
 sys.path.insert(0, '.')
 
 from dmscripts.helpers.auth_helpers import get_auth_token
-from dmscripts.bulk_upload_documents import get_bucket_name, get_all_files_of_type
+from dmscripts.bulk_upload_documents import get_all_files_of_type
```

Comment:
While we're at it can we move this function as well?

Reply:
Why not. Will add a separate commit